app/models/expenditure_report.rb

```diff
@@ -34,4 +35,15 @@ def blank_fund?
   def set_output_file
     self[:output_file] = output_file
   end
+
+  def start_date_present?
+    case date_type
+    when 'calendar'
+      cal_start.present?
+    when 'fiscal'
+      fy_start.present?
+    when 'paydate'
+      pd_start.present?
+    end
```

Comment:
Should we add a default date instead of making the user select a start date?

Reply:
My concern with adding a default start date is that they may not readily know what date that is, and may be unclear about the report they get. If they have to set it, then they will know. Also, it's not something they asked for.